code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -261,7 +259,7 @@
 				admin_inject_log(user, target, src, contained, reagents.get_temperature(), trans)
 			else
 				trans = reagents.trans_to(target, amount_per_transfer_from_this)
-			to_chat(user, SPAN_NOTICE("You inject [trans] units of the solution. The syringe now contains [src.reagents.total_volume] units."))
+			user.visible_message(SPAN_WARNING("[user] injects [target] with the syringe!"),SPAN_NOTICE("You inject [trans] units of the solution. The syringe now contains [src.reagents.total_volume] units."),2)
```

Comment:
Our reagent container might not necessarily be named a syringe. Use `with [src]` instead.